spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -396,7 +396,7 @@ def logger
     context 'on el-6' do
       let(:platform) { Beaker::Platform.new('el-6-i386') }
       it 'installs' do
-        expect(subject).to receive(:on).with(hosts[0], /puppetlabs-release-el-6\.noarch\.rpm/)
+        expect(hosts[0]).to receive(:install_package_with_rpm).with('http://yum.puppetlabs.com/puppetlabs-release-el-6.noarch.rpm', '--replacepkgs', {:package_proxy=>false})
```

Comment:
since we're not actually testing the entire URL that's being called, but just the form of the package name itself, it would be better to leave the regex as it was, rather than being so specific about it.  Let me know if this isn't the case for some reason, and we'll continue to talk about this.